api/src/main/java/org/openmrs/module/referencemetadata/RolePrivilegeMetadata.java

```diff
@@ -144,16 +148,97 @@ public void install() throws Exception {
         install(role(_Role.APPLICATION_HAS_SUPERUSER_PRIVILEGES, "Extends the underlying System Developer API role",
                 idSet(RoleConstants.SUPERUSER), idSet()));
 
-        install(role(_Role.APPLICATION_APPOINTMENT_VIEWER,
-                "Gives user the ability to view appointment schedules (but not to modify them)", idSet(), idSet(
-                _Privilege.APP_APPOINTMENTSCHEDULINGUI_HOME,
-                "App: appointmentschedulingui.viewAppointments")));
-
         install(role(_Role.APPLICATION_EDITS_EXISTING_ENCOUNTERS,
                 "Gives user the ability to edit patient encounter", idSet(), idSet(
                         _Privilege.TASK_EMR_PATIENT_ENCOUNTER_DELETE,
                         _Privilege.TASK_EMR_PATIENT_ENCOUNTER_EDIT)));
 
+        Module appointmentscheduling = ModuleFactory.getModuleById("appointmentscheduling");
+        Module appointmentschedulingui = ModuleFactory.getModuleById("appointmentschedulingui");
+        boolean isAppointmentModuleInstalled = (appointmentscheduling != null && appointmentschedulingui != null);
```

Comment:
Would it be clearer if isAppointmentModuleInstalled were a method of its own?

Reply:
And when you say `isAppointmentModuleInstalled`, do you mean the appointmentscheduling module or the appointmentschedulingui module?

Reply:
Since ModuleFactory.getModuleById gets looks at the loaded modules list, what happens when the module is loaded but not started?

Reply:
> Would it be clearer if isAppointmentModuleInstalled were a method of its own?

This can be done. Makes sense to me.

> And when you say `isAppointmentModuleInstalled`, do you mean the appointmentscheduling module or the appointmentschedulingui module?

I mean both, I can rename it to `areAppointmentModulesInstalled`.

> Since ModuleFactory.getModuleById gets looks at the loaded modules list, what happens when the module is loaded but not started?

If the modules are present and not started, it will go ahead and try to create roles. IMO, It will fail anyway if the module is present and not started.

I will make the first two changes, let me know if the third one is required.

Reply:
@dkayiwa Added the changes as mentioned above.
Changes were done as an amend to the existing commit for `RolePrivilegeMetadata`.